elb.tf

```diff
@@ -13,10 +13,10 @@ resource "aws_security_group" "acme_elb" {
     from_port   = 80
     to_port     = 80
```

Comment:
**Accurics remediation**
 
**[HIGH]: http port open to internet**
 
**Vulnerability**
http port open to internet
 
**Remediation**
Change cidr_block to non 0.0.0.0/0